frame/staking-rewards/src/lib.rs

```diff
@@ -308,9 +308,21 @@ pub mod pallet {
 		#[pallet::constant]
 		type PicaAssetId: Get<Self::AssetId>;
 
+		#[pallet::constant]
+		type XPicaAssetId: Get<Self::AssetId>;
```

Comment:
Not something we need to address in this PR, but we need to think of how to remove runtime specificity from the pallet

Reply:
I will move this conversation to slack so the PR can be merged